README.md

```diff
@@ -1 +1,54 @@
-# feast-custom-offline-store-demo
\ No newline at end of file
+# Feast Custom Offline Store
+[![test-offline-store](https://github.com/feast-dev/feast-custom-offline-store-demo/actions/workflows/test_custom_offline_store.yml/badge.svg?branch=main)](https://github.com/feast-dev/feast-custom-offline-store-demo/actions/workflows/test_custom_offline_store.yml)
+
+### Overview
+
+This repository demonstrates how developers can create their own custom `offline store`s for Feast.
+Custom offline stores allow users to use any underlying data store to as the source of truth for features which get materialized into the online feature store. Features can be retrieved from the offline store for model training. 
+
+
+### Why create a custom offline store?
+
+Feast uses an offline store as the source of truth for features. These features can be retrieved from the offline store for model training. Typically, scalable Data Warehouses are used for this purpose.
```

Comment:
Should `Data Warehouses` be capitalized?

Reply:
Sure